writeup/src/body.lhs

```diff
@@ -1662,6 +1673,9 @@ meaning that applying a patch over something that has been modified by
 this very patch amounts to not changing anything. The |patchIden| functions
 checks whether all changes in that patch are copies and |patchEquiv|
 checks if two patches are $\alpha$-equivalent.
+%Wouter: I wouldn't talk about laws of residual theory, but simply refer to the code
+%and the definition of disjoint a few lines above: that is---one of the patches
+%must be the identity or they are equal.
```

Comment:
I like mentioning residual laws. It shows we have thought of residuals and it is a good justification of why
can we do that.